javascript/builtins/Intl/PluralRules.json

```
@@ -302,7 +305,7 @@
                     "webview_ios": "mirror"
                   },
                   "status": {
-                    "experimental": true,
+                    "experimental": false,
```

Was this autofixed? If Chrome and Safari do not support this, it should still be marked as experimental.

```suggestion
                    "experimental": true,
```

---

yeah https://github.com/mdn/browser-compat-data/actions/runs/20098995930/job/57664598662#step:5:18

---

I'll look into fixing the experimental linter.

---

Filed https://github.com/openwebdocs/mdn-bcd-collector/issues/2936.